src/components/Organisms/Todocard/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import AddTaskButton from "../../Atoms/AddTaskButton";
+import Task from "../../Molecules/Task";
+
+const TodoCard = () => {
+  const [tasklist, setTasklist] = useState([]);
+  const onAddTaskButtonClick = () => {
+    const newTask = {
+      name: "",
+      initializing: true,
+    };
+    setTasklist(tasklist.concat(newTask));
+  };
+  const onTaskComplete = (index) => {
+    let newTasklist = tasklist.filter((_, i) => i !== index);
+    setTasklist(newTasklist);
```

Comment:
```suggestion
    const newTasklist = tasklist.filter((_, i) => i !== index);
    setTasklist(newTasklist);
```
再代入しない場合は `const` を使うほうが安全なコードになります！